Tests/Core/BasicTests.swift

```diff
@@ -62,7 +62,10 @@ class BasicTests: OperationTests {
         var finalCompletionBlockDidRun = 0
         operation.addCompletionBlock {
             finalCompletionBlockDidRun += 1
-            expectation.fulfill()
+            dispatch_async(Queue.Main.queue, {
+                guard let expectation = expectation else { print("Test: \(#function): Finished expectation after timeout"); return }
+                expectation.fulfill()
+            })
```

Comment:
This is all quite good, I think ideally all of these expectation `fulfill()` should be done in `OperationTests` using the `waitForOperation()` API.

It can be revisited when we move things across to _ProcedureKit_ and Swift 3.0.